runtime/jvmti/jvmtiThread.c

```diff
@@ -1324,8 +1324,11 @@ jvmtiGetCurrentThread(jvmtiEnv *env,
 static jvmtiIterationControl
 jvmtiSuspendResumeCallBack(J9VMThread *vmThread, J9MM_IterateObjectDescriptor *object, void *userData)
 {
-	j9object_t vthread = J9VMJDKINTERNALVMCONTINUATION_VTHREAD(vmThread, object->object);
-	if (NULL != vthread) {
+	j9object_t continuationObj = object->object;
+	j9object_t vthread = J9VMJDKINTERNALVMCONTINUATION_VTHREAD(vmThread, continuationObj);
+	ContinuationState continuationState = J9VMJDKINTERNALVMCONTINUATION_STATE(vmThread, continuationObj);;
+
+	if ((NULL != vthread) && J9_ARE_NO_BITS_SET(continuationState, J9_GC_CONTINUATION_STATE_LAST_UNMOUNT)) {
```

Comment:
@fengxue-IS This source code is in C; we can't invoke C++ code from `ContinuationHelpers.hpp` here; so, I have employed a workaround.